src/components/BynderInput.tsx

```diff
@@ -104,14 +113,16 @@ const BynderInput = (props: Props) => {
           sources={[{ src: value.previewUrl }]}
         />
       );
-    } else {
+    }
+    if (value.type === 'IMAGE') {
```

Comment:
in blocks we just get an image with no source that just says "preview" - ideally we can add an empty state in here. is there something in sanity UI?